### DISCUSSION TITLE
Multiple Devices

### DISCUSSION THREAD

**Discussion by nicobezz** [danielrivard/homeassistant-innova#231] [Q&A] [answered]
Hi,

Thanks for the great integration! 

I’m encountering a small issue: when I try to add multiple devices by their IP addresses, the integration recognizes all of them, but only one shows up as a device in the interface. The others aren't being recognized as devices by HA. How can I get all my devices to appear?

**Comment by jtome** [comment]
I have 7 devices and all of them are recognized as different devices by HA...
<img width="750" alt="screenshot02" src="https://github.com/user-attachments/assets/3276fc15-5c2d-46fe-b98e-2b490fd85cf5">

 My actual installation is...
<img width="457" alt="screenshot01" src="https://github.com/user-attachments/assets/dc5af209-07da-4539-a1f7-005e081bf492">

But I have been suffering multiple problems about devices randomly disappearing, of switching off without logical reason. After fighting with this problems for some days I found that the problem was with the propietary mobile application: InnovApp AllinOne, that I used to do the initial configuration of the devices. I have to ensure that in the application all the devices was in manual mode (instead of Automatic/Programmed). After changing all the devices to manual mode they don't have had more problems.

Perhaps this could be also your problem. Good luck.

Jorge

**Comment by nicobezz** [comment]
My problem is that they don't even show up as devices. Only the first one that I have configured works. Maybe I'm missing a step or doing something wrong.
<img width="1042" alt="image" src="https://github.com/user-attachments/assets/230a53ce-c1f4-4b40-8134-56fe6963d6b1">

**Reply by jtome** [reply]
The process is pretty straightforward, you only have to introduce the IP address of the device and it should work. You can check if the device is really available if you try to access using a browser to the URL http://192.168.1.xx/api/v/1/status (replace 'xx' by the correct one in your environment)

**Reply by nicobezz** [reply]
That's exactly what I did... The Ip addresses are correct, when doing POST commands manually the ACs respond.

**Comment by danielrivard** [comment, marked as answer]
Hi @nicobezz, your problem is probably related to the serial number of the devices, they probably all share the same one and the last one you setup is probably the only one that is kept. The serial number is the id that the integration stores in HA's database to distinguish between them.

See this answer from a similar issue.
https://github.com/danielrivard/homeassistant-innova/issues/187#issuecomment-2040065106

**Reply by nicobezz** [reply]
Thanks! I didn't see the previous threads.
